content/guides/embed/ui-elements/design-system-tokens.md

```diff
@@ -0,0 +1,192 @@
+---
+rank: 12
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/theming-styling
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+
+# Design system tokens
```

Comment:
Maybe we could name this page UI Elements design tokens? Just for people how are accessing the page via link to give more context